content/docs/reference/pkg/kubernetes/helm/v2/chart.md

```diff
@@ -113,7 +113,7 @@ class HelmStack : Stack
             Version = "1.24.4",
             FetchOptions = new ChartFetchArgs
             {
-                Repo = "https://kubernetes-charts.storage.googleapis.com/"
```

Comment:
Ordinarily we wouldn't make changes to these files, since they're generated -- but since the file that's used to generate them is included in this PR as well, and the generation process relies on this repo, I can't think of a reason not to update the generated files here, too. 😄 Mainly just calling it out for your info.

Reply:
good to know